### ISSUE TITLE
subxt build error about rpc

### ISSUE THREAD

**Issue by Lawliet-Chan** [paritytech/substrate-subxt#132]
Hi, I use subxt-v0.8.0 and subxt-v0.9.0, they both have some bugs as follows:
```
 Compiling substrate-subxt v0.9.0
error[E0107]: wrong number of type arguments: expected 0, found 1
  --> /Users/lawliet/.cargo/registry/src/crates.rustcc.cn-46a557d00542c05f/substrate-subxt-0.9.0/src/rpc.rs:89:26
   |
89 | impl<T> From<NumberOrHex<<T as System>::BlockNumber>> for BlockNumber<T>
   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^ unexpected type argument

error[E0107]: wrong number of type arguments: expected 0, found 1
  --> /Users/lawliet/.cargo/registry/src/crates.rustcc.cn-46a557d00542c05f/substrate-subxt-0.9.0/src/rpc.rs:87:47
   |
87 | pub struct BlockNumber<T: System>(NumberOrHex<<T as System>::BlockNumber>);
   |                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^ unexpected type argument

error[E0107]: wrong number of type arguments: expected 0, found 1
  --> /Users/lawliet/.cargo/registry/src/crates.rustcc.cn-46a557d00542c05f/substrate-subxt-0.9.0/src/rpc.rs:93:28
   |
93 |     fn from(x: NumberOrHex<<T as System>::BlockNumber>) -> Self {
   |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^ unexpected type argument

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0107`.
error: could not compile `substrate-subxt`.
```
My rust toolchain is `nightly-2020-06-16`

**Comment by ascjones** [comment]
I've just released `v0.10.0`. Let me know whether this fixes it for you.

**Comment by Lawliet-Chan** [comment]
It works.
